### ISSUE TITLE
A11Y: Image question

### ISSUE THREAD

**Issue by OlgaLarina** [surveyjs/survey-library#9845]
https://surveyjs.io/form-library/examples/add-image-and-video-to-survey/reactjs

- [x] Frames must have an accessible name. Ensure iframe and frame elements have an accessible name
- [x] Ensure ARIA attributes are not prohibited for an element's role (aria-label attribute cannot be used on a div with no valid role attribute.)
